hyp3_metadata/templates/insar/readme.md.txt.j2

```diff
@@ -0,0 +1,298 @@
+ - offset_fit      # Range and azimuth offset polynomial estimation
+ - SLC_diff_intf   # Differential interferogram generation from co-registered SLCs and a simulated interferogram
+ - offset_add      # Add range and azimuth offset polynomial coefficients
+
+### Check for Convergence
+
+ Azimuth offset cannot exceed 0.02 pixels
+
+### Perform Enhanced Spectral Diversity Refinement
+
+ - S1_coreg_overlap   # Determine co-registration offset based on the burst overlap
+
+### Perform Final Matching
+
+ - SLC_interp_lt_S1_TOPS # Resample TOPS (IW mode) SLC using a lookup table and SLC offset polynomials for refinement
+ - create_offset   # Create and update ISP offset and interferogram parameter files
+ - offset_pwr      # Offset estimation between SLC images using intensity cross-correlation
+ - offset_fit      # Range and azimuth offset polynomial estimation
+ - SLC_diff_intf   # Differential interferogram generation from co-registered SLCs and a simulated interferogram
+
+## Unwrapping and Geocoding ##
+
+### Unwrapping
+
+ - adf          # Adaptive spectral filtering for complex interferograms
+ - rasmph_pwr   # Generate 8-bit raster graphics image of the phase of complex data combined with intensity data
+ - rascc        # Generate 8-bit raster graphics image of correlation coefficient
+ - rascc_mask   # Generate phase unwrapping validity mask using correlation and intensity
+ - mcf          # Phase unwrapping using Minimum Cost Flow (MCF) and triangulation
+ - rasrmg       # Generate 8-bit raster graphics image from unwrapped phase and intensity data
+ - dispmap      # Conversion of unwrapped differential phase to displacement map (m) - Line-Of-Sight
+ - dispmap      # Conversion of unwrapped differential phase to displacement map (m) - Vertical
+
+### Post-Processing
+
+ - geocode_back   # Geocoding of image data using lookup table values
+ - data2geotiff   # Convert geocoded data with DEM parameter file to GeoTIFF format
+
+*************
+# The Sentinel-1 mission #
+
+The Sentinel-1 satellite constellation is part of the Copernicus Earth Observation program, coordinated by the European Space Agency (ESA) on behalf of the European Commission (EC). Sentinel-1 satellites carry C-band Synthetic Aperture Radar (SAR) instruments for global, around-the-clock imagery, even through cloud cover. The Sentinel-1A satellite was launched April 3, 2014, and the Sentinel-1B satellite was launched April 25, 2016. The satellites each have a 12-day repeat cycle, but are offset 180 degrees in order to repeat passes every 6 days.
+
+More information about the mission is available at:
+https://earth.esa.int/web/guest/missions/esa-operational-eo-missions/sentinel-1
+
+Additional information about Sentinel-1 data, imagery, tools and applications is available at:
+https://asf.alaska.edu/data-sets/sar-data-sets/sentinel-1
+
+*************
+For assistance, contact the Alaska Satellite Facility:
+uso@asf.alaska.edu
+907-474-5041
```

Comment:
Should we add a link for Gitter?

Reply:
We have a dedicated "Contact Us" page on the HyP3 website now, that might be the best place to direct users: https://hyp3-docs.asf.alaska.edu/contact/

Reply:
Added HyP3 contact link